src/Assets/TestProjects/SampleManifest/Sample.json

```diff
@@ -151,6 +152,13 @@
                 "win-x64": "Mono.Android.LLVM.Aot.armv7a.Win64Host"
             }
         },
+        "Mono.Android.Sdk.Extra": {
+            "version": "8.4.7",
+            "kind": "sdk",
+            "alias-to": {
+                "foo": "Mono.Android.Sdk.Extra.foo"
```

Comment:
Adding a fake pack with no appropriate alias to the manifest used for testing, this causes the issue to repro in several existing tests.

Reply:
Is that an authoring error? why it points to a non existing pack?

Reply:
This is simulating a pack that doesn't have a alias for the current rid, except for testing purposes it's helpful for this pack to not have a valid alias for any rid's.